test/runtests.jl

```diff
@@ -2,7 +2,7 @@ using Aqua, CpuId, ReTestItems, ExponentialFamily
 
 # `ambiguities = false` - there are quite some ambiguities, but these should be normal and should not be encountered under normal circumstances
 # `piracy = false` - we extend/add some of the methods to the objects defined in the Distributions.jl
-Aqua.test_all(ExponentialFamily, ambiguities = false, piracy = false)
+Aqua.test_all(ExponentialFamily, ambiguities = false, deps_compat = false, piracy = false)
```

Comment:
Please do not disable essential tests just for the sake of a green test
```suggestion
Aqua.test_all(ExponentialFamily, ambiguities = false, deps_compat = (; check_extras = false, check_weakdeps = true), piracy = false)
```